PrioritiesSprint4.md

```diff
@@ -0,0 +1,92 @@
+These are the priorities for Sprint number 4, which runs from June 17, 2024 to July 19, 2024. 
+
+In this sprint, the builds are in these environments: 
+
+- PROD: Octopus
+- TEST: Eel 
+- CI: Fugu
+- DEV: Guppy
+
+Each priority has a list of subtasks that are to be completed within this sprint.
+
+Note that ARAs will be required to implement at least one of MCQ, Pathfinder, or LLM.
+
+# Develop Pathfinder Query
+## Pre-development work:
+- Develop automated tests
+## Fugu (CI) work:
+- Implement Constraints
+- Improve Results
+- Evaluate walltime performance and set goals
+
+# MultiCURIE Query (MCQ)
+## Pre-development work:
+- Develop automated tests
+- Develop Drug Side Effect Use Case
+## Fugu (CI):
+- ARA Implementation of Creative Mode
+- Improve KP results for Phenotypes to Disease
+- Implement Set IDs in NodeNorm
+
+# LLM Summarization 
+## Pre-development work:
+- Experiment with and refine the summarization idea: Decide whether we will implement single-answer summarization, all-answers summarization, user querying over answers (RAG-like), or other approaches.
+
+# O&O Based grouping in the UI 
+## Pre-development work:
+- User testing to determine effectiveness of chemical grouping based on ATC and Chebi roles
+- User testing to determine effectiveness of gene grouping based on Genetics KP Gene Sets and PFOCR
+## Fugu (CI):
```

Comment:
Per Marc Duby on 7/2, - Implementation of Gene Set grouping API (Genetics KP) milestone will be in their Dev environment by 7/19 but not in CI.